code/headers/string.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <cstdint>
+
+namespace r2d2::string {
```

Comment:
Please only use your own namespace. If these functions actually have value for other modules, a library should be extracted instead.

Reply:
should be fixed in #8